src/graphql/schemas/typeDefs/hyperboardTypeDefs.ts

```diff
@@ -0,0 +1,135 @@
+import { Field, ObjectType } from "type-graphql";
+import { BasicTypeDef } from "./baseTypes/basicTypeDef.js";
+import { EthBigInt } from "../../scalars/ethBigInt.js";
+import { User } from "./userTypeDefs.js";
+import { GraphQLBigInt } from "graphql-scalars";
+
+@ObjectType({
+  description: "Hyperboard of hypercerts for reference and display purposes",
+})
+class Hyperboard extends BasicTypeDef {
+  @Field({ description: "Name of the collection" })
+  name?: string;
+  @Field(() => [EthBigInt], {
+    nullable: true,
+    description: "Chain ID of the hyperboard",
+  })
+  chain_ids?: (bigint | number)[];
+  @Field({ nullable: true, description: "Background image of the hyperboard" })
+  background_image?: string;
+  @Field({
+    nullable: true,
+    description:
+      "Whether the hyperboard should be rendered as a grayscale image",
+  })
+  grayscale_images?: boolean;
+  @Field({
+    nullable: true,
+    description: "Color of the borders of the hyperboard",
+  })
+  tile_border_color?: string;
+
+  @Field(() => [User])
+  admins?: User[];
+
+  @Field(() => SectionWrapper)
+  sections?: SectionWrapper[];
+}
+
+@ObjectType({})
+class SectionWrapper {
+  @Field(() => [Section])
+  data?: Section[];
+
+  @Field()
+  count?: number;
+}
+
+@ObjectType({
+  description: "Collection of hypercerts for reference and display purposes",
+})
+class Collection extends BasicTypeDef {
+  @Field({ description: "ID of the collection" })
+  created_at?: string;
+  @Field({ description: "Name of the collection" })
+  name?: string;
+  @Field({ description: "Description of the collection" })
+  description?: string;
+  @Field({ description: "Whether the collection should be hidden" })
```

Comment:
Hidden from? Non-admins? All users? Will it still be queryable?

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/commit/f1f11c9371862470da7cc49cc3cdf254ed48d1b8 by hiding it for now, as I'm not sure we'll be using this going forward.